File: library/src/main/java/pl/aprilapps/easyphotopicker/DefaultCallback.java
package pl.aprilapps.easyphotopicker;

/**
 * Stas Parshin
 * 05 November 2015
 */
public abstract class DefaultCallback implements EasyImage.Callbacks {

    @Override
    public void onImagePickerError(Exception e, EasyImage.ImageSource source, int type) {
    }

    @Override
    public void onCanceled(EasyImage.ImageSource source, int type) {
    }
}

File: library/src/main/java/pl/aprilapps/easyphotopicker/DefaultCallback.kt
package pl.aprilapps.easyphotopicker

/**
 * Stas Parshin
 * 05 November 2015
 */
abstract class DefaultCallback : EasyPicker.Callbacks {

    override fun onPickerError(error: Throwable, source: MediaSource) {}

    override fun onCanceled(source: MediaSource) {}
}
